src/Repository/LikesRepository.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repository;
+
+use App\Library\PDO;
+use App\Entity\LikesEntity;
+
+class LikesRepository
+{
+	//like.php.45
+	public static function setLikeByAll(int $id, string $username, int $post_id): LikesEntity
```

Comment:
Again, please keep more meaningful naming.

Please try to not make references to usage of method like `//like.php.45` -> every modern IDE is able to handle it for you.